h2o-core/src/main/java/water/api/RequestServer.java

```diff
@@ -877,39 +870,40 @@ private static String getOutputLogStem() {
     return "h2ologs_" + now;
   }
 
-  private static byte[] zipLogs(byte[][] results, byte[] clientResult, String topDir) throws IOException {
+  private static byte[] archiveLogs(LogArchiveContainer container, Date now,
+                                    byte[][] results, byte[] clientResult, String topDir) throws IOException {
     int l = 0;
     assert H2O.CLOUD._memary.length == results.length : "Unexpected change in the cloud!";
     for (byte[] result : results) l += result.length;
     ByteArrayOutputStream baos = new ByteArrayOutputStream(l);
 
     // Add top-level directory.
-    ZipOutputStream zos = new ZipOutputStream(baos);
+    LogArchiveWriter zos = container.createLogArchiveWriter(baos);
```

Comment:
`zos` implies ZipOutputStream but we use `LogArchiveWriter`

Reply:
right, thx